tracking-migration/src/main/java/io/customer/tracking/migration/util/JsonAdapter.kt

```diff
@@ -41,7 +41,7 @@ class JsonAdapter {
         val taskJson = runCatching { JSONObject(data) }.getOrElse {
             throw RuntimeException("Queue task data is invalid for $task. Could not run task.")
         }
-        val timestamp = taskJson.longOrNull("timestamp") ?: Date().time
+        val timestamp = taskJson.longOrNull("timestamp") ?: Date().getUnixTimestamp()
```

Comment:
is this because queue keep them in seconds or server expects them in seconds?

Reply:
It is because that's how we [parse Date in JsonAdapter in `main` today](https://github.com/customerio/customerio-android/blob/f1c1ceedfb053d04a8e4c2d2f6e26ccbb84a1e29/sdk/src/main/java/io/customer/sdk/data/moshi/adapter/UnixDateAdapter.kt#L31)